src/services/identity/agent.ts

```diff
@@ -418,14 +424,6 @@ export class Veramo {
 		return await agent.cheqdRevokeCredential({ credential: credentials, fetchList: true, publish });
 	}
 
-	async resolve(didUrl: string) {
-		const result = await fetch((process.env.RESOLVER_URL || DefaultResolverUrl) + didUrl, {
-			headers: { 'Content-Type': 'application/did+ld+json' },
-		});
-		const ddo = await result.json();
-		return ddo;
-	}
```

Comment:
Isn't this header still required? Unless it's accounted for somewhere else. I'd also probably do it as `'application/did+ld+json', '*/*'`: the first is relevant if it's a DID Document being returned, but if you're resolving resources or other things, you need the catch-all header as well. Otherwise, the resolver should theoretically reject it (I don't know if it actually does).

Reply:
Added `Content-Type: '*/*'` since that `resolve` method uses for DID Resolution and DID Dereferencing.